actix-session/src/lib.rs

```diff
@@ -180,6 +180,10 @@ impl Session {
         }
     }
 
+    /// Adds the given key-value pairs to the session on the request.
+    ///
+    /// Values that match keys already existing on the session will be overwritten. The values in
+    /// the key-value pairs should be serialized into JSON.
```

Comment:
Maybe: `Keys and values should already be JSON serialized.` reads a bit clearer.

Reply:
Good suggestion. I updated the documentation. Note that only the value needs to be serialized.